spec/factories/company_branches.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+    factory :company_branch do
+        name { 'New york city' }
+        address { Faker::Address.full_address }
+    end
+end
```

Comment:
add an extra empty line at the end of the file here and in the employees `factory`

Reply:
Thanks.

Why need to add a extra line?, I never understand that.

About remove test, we remove the default dummy test generated by rails with minitest, I removed it because we change framework to rspec. The files removed didn't work.

Reply:
https://thoughtbot.com/blog/no-newline-at-end-of-file#history-lesson for te NewLine thing.

Reply:
Yes we need controller test, but we change de framework from minitest to RSpec and we didn't code it. we will code that soon.

Reply:
We made the changes. Thx @dannharo